### ISSUE TITLE
Respect load flag during export

### ISSUE THREAD

**Issue by tilmankamp** [mozilla/DeepSpeech#2722]
During model export the `--load` flag is not respected. This leads to unexpected outcomes and/or requires users to manually edit the checkpoint file if they want to export a different model than the most recent one.

**Comment by tilmankamp** [comment]
On hold till #2696 lands...

**Comment by Jendker** [comment]
But this is the case only on current master, right? Does it work as expected on 0.6.1?

**Comment by reuben** [comment]
Use the source, Luke: https://github.com/mozilla/DeepSpeech/blob/v0.6.1/DeepSpeech.py#L783

**Comment by lissyx** [comment]
> On hold till #2696 lands...

Are we waiting on #2696 to **fix** it or just to do the fix ?

**Comment by tilmankamp** [comment]
This one here is on hold as it should use the `try_model` helper function of #2696 .

**Comment by reuben** [comment]
Fixed in #2763.
